src/main/java/com/google/sps/servlets/ElectionServlet.java

```diff
@@ -49,11 +62,20 @@
   private static final String baseURL = "https://www.googleapis.com/civicinfo/v2/elections?key=";
   private static final Logger logger = Logger.getLogger(ElectionServlet.class.getName());
 
-  // TODO(anooshree): Change GET to PUT and store retrieved information in the database
   @Override
-  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+  public void doPut(HttpServletRequest request, HttpServletResponse response) throws IOException {
+
+        // Deleting the queries from yesterday in the case that they are outdated
```

Comment:
Could we update them (e.g. by Election#id) instead of deleting them?

Reply:
Sure -- I did this to handle the case where elections that were present the previous day are no longer available. Because Datastore access entities by ID and not by index, we wouldn't know the difference in the number of elections between days and would thus have to compare the entries side by side, so I figured deleting and repopulating the entities would be faster.

Reply:
I see.  Yeah I think that approach is probably ok for now.  We can always change it later if there are issues.

Re elections that have passed, I think the frontend/ client can always just filter out elections by election day as well.